server/migrations/versions/c78e97acf44d_.py

```
@@ -0,0 +1,122 @@
+    sa.Column('name', sa.String(length=100), nullable=False),
+    sa.Column('contact', sa.String(length=20), nullable=False),
+    sa.Column('email', sa.String(length=100), nullable=False),
+    sa.Column('date_joined', sa.DateTime(), nullable=True),
+    sa.PrimaryKeyConstraint('id'),
+    sa.UniqueConstraint('contact'),
+    sa.UniqueConstraint('email')
+    )
+    op.create_table('services',
+    sa.Column('id', sa.Integer(), nullable=False),
+    sa.Column('name', sa.String(length=100), nullable=False),
+    sa.Column('price', sa.Integer(), nullable=False),
```

Flake8 (7.2.0)

[error] continuation line missing indentation or outdented

(E122)